packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -730,7 +495,7 @@ export class GarbageCollector implements IGarbageCollector {
 			// Log an event so we can evaluate how often we run into this scenario.
 			logger.sendErrorEvent({
 				eventName: "CollectGarbageCalledWithoutTimestamp",
-				gcConfigs: JSON.stringify(this.configs),
+				details: JSON.stringify({ ...this.configs }),
```

Comment:
revert

Reply:
And in refreshLatestSummary, I'd rename `details` to `gcConfigs` for consistency/clarity